torch/_dynamo/eval_frame.py

```diff
@@ -356,21 +356,31 @@ def get_compiler_config():
         fn = innermost_fn(fn)
 
         # add context containing GraphModule to any GraphModule forward functions
-        if isinstance(fn, torch.fx.GraphModule):
```

Comment:
Why are we removing this case? It seems like `test_preserve_fx_node_metadata` is failing as a result.

Reply:
Didn't realize some code is ran for non _LazyGraphModule in this code block. Fixing it now.